Testing/xASL_test_IntegrationTesting.m

```diff
@@ -109,14 +116,30 @@
 
 
 %% Run individual flavor dataset
-function xFlavor = xASL_test_thisFlavor(testingRoot, databaseRoot, flavorName)
+function xFlavor = xASL_test_thisFlavor(testConfig, flavorName)
 
+    % Get paths
+    testingRoot = testConfig.pathTest;
+    databaseRoot = fullfile(testConfig.pathTest,'FlavorDatabase');
+    
     % Copy flavor
     fprintf('Copy %s...\n', flavorName);
     xASL_Copy(fullfile(databaseRoot,flavorName),fullfile(testingRoot,'FlavorDatabaseTest',flavorName),1);
     
     % Remove rawdata folder from testing dataset
     xASL_delete(fullfile(testingRoot,'FlavorDatabaseTest',flavorName,'rawdata'),1);
+    
+    % Create default dataPar.json
+    defaultDataPar.x.subject_regexp = '^sub-.*$';
```

Comment:
Why did you add these here, when they are defaulted by ExploreASL itself? It seems unnecessary convoluted for me now, you can probably remove many testing lines and make the scripts more efficient and readable?

Reply:
We need ...
```
    defaultDataPar.x.bUseMNIasDummyStructural = 1;
    defaultDataPar.x.Quality = 0;
```
... those are not default.

Reply:
237f78d